aws_jobs/cyhy-data-extract.py

```diff
@@ -390,6 +390,8 @@ def main():
                 "agency.location": True,
                 "agency.name": True,
                 "agency.type": True,
+                "agency.contacts": True,
+                "key": True,
```

Comment:
The key is sensitive data, which is why it was never exported in the past.  Please provide confirmation via email that the CyHy data owner and the owner of the receiving system (the Analytics Environment) understand and accept the risks of sharing this data.

Reply:
With https://github.com/cisagov/cyhy-feeds/pull/52/commits/e1f73a9d5398388127b376820c50e66bf0f21a18, the `key` field is no longer explicitly mentioned, but the concern above still stands.

Reply:
I've reached out to get that approval

Reply:
@DJensen94 Do you have any updates on this?